source/docs/flow_control.md

```diff
@@ -8,6 +8,30 @@ buffer limits in Envoy are considered soft limits. When the buffer eventually dr
 half of the high watermark to avoid thrashing back and forth) the low watermark callback will
 fire, informing the sender it can resume sending data.
 
+### Configuration
```

Comment:
It would really be fantastic to just get this directly into a FAQ entry in the main docs, as I think it would immediately help folks and I think discoverability of this page is low. With that said, if you don't have time to do that now, this is great.

Reply:
Do you think we should add a FAQ entry pointing here, or move this section to the FAQ?

Reply:
My preference is to move the whole section to the FAQ, because from there you can directly ref link to the relevant config fields, etc.

Reply:
SG - I added a FAQ doc so I'll just revert the changes to this file